### ISSUE TITLE
Come up with a decent design for the service details page

### ISSUE THREAD

**Issue by iloire** [iloire/WatchMen#26]
Any takers? :)

**Comment by iloire** [comment]
Some inspiration for bootstrap:
https://wrapbootstrap.com/themes/admin

**Comment by iloire** [comment]
I actually love this one: http://demo.kimlabs.com/gentelella/production/index.html

**Comment by mvdkleijn** [comment]
Personally I think the interface is fine as is... unless you are planning to add a load of new features and admin options/graphs?

**Comment by gh0st** [comment]
I agree with @mvdkleijn. I'd be willing to put some time in if there are a load of new features and admin options/graphs.
